core/src/main/java/hudson/model/ViewGroupMixIn.java

```diff
@@ -99,6 +101,15 @@ public View getView(String name) {
             View pv = getPrimaryView();
             if (pv instanceof ViewGroup)
                 return ((ViewGroup)pv).getView(name);
+            if (pv instanceof AllView && AllView.DEFAULT_VIEW_NAME.equals(pv.name)) {
+                // JENKINS-38606: primary view is the default AllView, is somebody using an old link to localized form?
+                for (Locale l : Locale.getAvailableLocales()) {
+                    if (name.equals(Messages._Hudson_ViewName().toString(l))) {
```

Comment:
So changing the translations will break this compatibility measure for that language?